src/components/ActivitiesListItem/ActivitiesListItem.js

```diff
@@ -74,7 +74,7 @@ ActivitiesListItem.defaultProps = {
   filter: '',
 };
 
-const mapStateToProps = ({ filter }) => ({
+const mapStateToProps = ({ activities: { filter } }) => ({
```

Comment:
this name is very problematic. When dev hears filter, he is thinking about filter method. I think, that it would be a nice improvement to change it somehow (maybe `filterKey` or something like that?)